sources/settings-extensions/ntp/src/lib.rs

```diff
@@ -8,6 +8,7 @@ use std::convert::Infallible;
 #[model(impl_default = true)]
 pub struct NtpSettingsV1 {
     time_servers: Vec<Url>,
+    options: Vec<str>,
```

Comment:
You'll need to use an owned string here:

```suggestion
    options: Vec<String>,
```